### ISSUE TITLE
How can I change some rule like `[whitespace/braces]`?

### ISSUE THREAD

**Issue by felixf4xu** [cpplint/cpplint#204]
Hi,

If I write code like this:
```c
int main()
{
  return 0;
}
```
I will have a cpplint error like this:
```
{ should almost always be at the end of the previous line  [whitespace/braces]
```
If I want to use `cpplint` but our code style is to put `{` in a new line (as in my code above), where should I change?

**Comment by tkruse** [comment]
cpplint is not so flexible to allow for different styles of braces, you could only deactivate this error category.

**Comment by correaa** [comment]
I also find this warning when using this type of idiom with manually defined scopes.

```
int main() {
{
  scope1;
}
{      // cpplint ask for an space before {
  scope2;
}

}
```

**Comment by tkruse** [comment]
Agreed, unnamed scopes sadly seem to not have been considered in the cpplint tool, so far. Sadly it's likely not easy to easily detect them as such in cpplint, as cpplint does not parse the syntax tree.

**Comment by jeissonh** [comment]
For OpenMP one-line directives before code blocks, cpplint works great, e.g:

```c
#pragma omp parallel num_threads(thread_count)
{
  // Concurrent instructions here
}
```

But if your pragma exceeds 80 chars and you slice it escaping the new line, e.g;

```c
#pragma omp parallel num_threads(thread_count) \
    default(none) shared(std::cout, thread_count)
{
  // Concurrent instructions here
}
```

you will get a cpplint diagnostic:

    your.cpp:3:  { should almost always be at the end of the previous line  [whitespace/braces]

How to fix it?

**Comment by aaronliu0130** [comment]
For now, you can do `// NOLINTLINE` on the line or add `-whitespace/braces` in the filter configuration. I recommend you file a new issue so we can track the progress of fixing it better.

**Comment by jeissonh** [comment]
Thank you aaronliu. I opened a new issue (#294).
